spec.html

```diff
@@ -26024,6 +26140,17 @@ <h1>Abstract Module Records</h1>
                 Purpose
               </th>
             </tr>
+            <tr>
+              <td>
+                GetModuleSource()
+              </td>
+              <td>
+                <p>It returns either a normal completion completion for the ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), or a throw completion.</p>
+                <p>When called multiple times on the same Module Record, if GetModuleSource() returns a normal completion it must always return a normal completion containing the same object.</p>
+                <p>The returned object should be an instance of a subclass of %AbstractModuleSource%, and it must have an internal slot [[ModuleSourceRecord]].</p>
+                <p>Module Record's that do not have a source represemtation must always return a throw completion containing a *ReferenceError*.</p>
```

Comment:
```suggestion
                <p>Module Records that do not have a source representation must always return a throw completion containing a *ReferenceError*.</p>
```

Reply:
We don't actually have a term for "throw completion containing". We can add it, or we could say "throw completion whose [[Value]] is a ReferenceError".

```suggestion
                <p>For Module Records that do not have a source representation, GetModuleSource must always return a throw completion whose [[Value]] is a *ReferenceError*.</p>
```

Reply:
This line was removed in the current spec.